spark-tensorflow/prepare/src/main/scala/com/google/cloud/ml/samples/criteo/CriteoIndexer.scala

```diff
@@ -68,15 +68,13 @@ trait CriteoIndexer {
     }).
       toMap
 
-  def transform(rawDf: DataFrame, resource: IndexerResource): DataFrame = {
+  def getEmbeddings(resource: IndexerResource): Map[String, DataFrame] = {
```

Comment:
As a solution to our problems with the term "embedding", let's just call them vocabularies (or ordered vocabularies)? That's consistent with the tf.feature_column names.

Reply:
done...i renamed embeddings to vocabularies and vocabularies to featurevaluecounts, that made sense to me, LMK what you think.